dashboard-frontend/src/types/study.ts

```diff
@@ -0,0 +1,33 @@
+export interface Study {
+  id: string;
+  name: string;
+  slug: string;
+  createdAt: string;
+  progress: StudyProgress;
+  strata: Strata[];
+}
+
+interface StudyProgress {
+  currentParticipants: number;
+  expectedParticipants: number;
+  currentAverageDeviation: number;
+  expectedAverageDeviation: number;
+  date: string;
+  averageDeviationOverTime: DataPoint[];
+}
+
+interface Strata {
+  id: string;
+  percentageDesired: number;
+  currentPercentage: number;
+  remainingPercentage: number;
+  expectedPercentage: number;
+  currentPriceToAdquireParticipant: number;
+  currentBudgetDecided: number;
```

Comment:
I would prefer "set" or "configured". It's not just that it decided the budget, it actually set it on facebook!

Reply:
Yes, makes sense 👍

Now that **StratumProgress** is a point/set of data in a specific datetime, what do you think about just calling it **currentBudget** ?